frontend/src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
           <Route path="/login" element={<Login />} />
           <Route path="/home" element={<Dashboard />} />
           <Route path="/profile" element={<Profile />} />
+          <Route path="/editProfile" element={<EditProfile />} />
```

Comment:
Maybe a good route would be:
```suggestion
          <Route path="/profile/edit" element={<EditProfile />} />
```